sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -358,6 +359,7 @@ object ExecutionPlanCaptureCallback {
   }
 
   def assertContains(gpuPlan: SparkPlan, className: String): Unit = {
+    implicit val regexMap = scala.collection.mutable.Map.empty[String, Try[Regex]]
```

Comment:
The use of implicit parameter passing seems unnecessary and makes the code harder to read.  It looks like everywhere that calls `containsPlan` now needs to make sure they create an implicit map val before calling, so I'm wondering why we don't just pass the map explicitly?  My Scala-fu is not the best, so maybe I'm missing something.